bin/opm

```diff
@@ -1509,7 +1510,14 @@ SKIP_INSTALL:
 }
 
 sub get_install_dir () {
-    if (defined $install_into_cwd) {
+    if (defined $install_into_dir) {
+        my $dir = File::Spec->rel2abs($install_into_dir);
+        if (!-d $dir) {
+            make_path $dir;
+        }
+        return $dir;
+    }
+    elsif (defined $install_into_cwd) {
```

Comment:
Because the previous branch always does `return`, let's replace `elsif` with `if` here.